wiki/BioSQL.md

```diff
@@ -84,26 +84,43 @@ Creating the empty database
 
 ### MySQL
 
-The following command line should create a new database on your own
+Some systems like Ubuntu, new version of mysql is using by default the [UNIX auth_socket plugin](https://dev.mysql.com/doc/mysql-security-excerpt/5.5/en/socket-pluggable-authentication.html) if the password is left empty while installing. To proceed, a password is recommended.
```

Comment:
This does not flow nicely...

Reply:
Can we omit this (how to set the password and restart the service), or point the user at relevant MySQL documentation?

Reply:
I've deleted these and referred to the MySQL documentation in the section of installation.